### PR TITLE
Decomposition

### PR DESCRIPTION
**Pull request by andylvua**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #include "Inc/modem/serial.hpp"
 #include "Inc/modem/modem.hpp"
 #include "Inc/cli/cli.hpp"
-#include "Inc/cli/colors.hpp"
+#include "Inc/cli/defenitions/colors.hpp"
```
Comment (codefloww) [inline main.cpp]: learn english pls

File: Src/cli/utils/consoles.cpp [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+        free((void *) ussd);
+    }
+
+    modem.disableUSSDConsoleMode();
+
+    initScreen();
+    renderScreen();
+}
+
+std::string parseUrl(std::string httpCommand) {
```
Comment (codefloww) [inline Src/cli/utils/consoles.cpp:93]: maybe should be in cli_utils file

File: Src/cli/utils/consoles.cpp [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    std::basic_regex<char> urlRegex(
+            "^(http|https)://[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)+([a-zA-Z0-9-._~:/?#[\\]@!$&'()*+,;=]*)?$"
+    );
+    if (!std::regex_match(url, urlRegex)) {
+        std::cout << RED_COLOR << "Invalid URL" << RESET << std::endl;
+        return "";
+    }
+
+    return url;
+}
```
Comment (codefloww) [inline Src/cli/utils/consoles.cpp:113]: maybe should be in cli_utils

**Review by codefloww — COMMENTED** [review]
Move Src/command to Src/modem as it's done in Inc. Name media_types may be misleading. Other changes look ok.

**Comment by andylvua** [comment]
> Move Src/command to Src/modem as it's done in Inc. Name media_types may be misleading. Other changes look ok.

Fixed in 635f79b

File: main.cpp (hunk 1 above)
Comment (andylvua) [reply main.cpp]: Fixed in 3f85561

File: Src/cli/utils/consoles.cpp (hunk 3 above)
Comment (andylvua) [reply Src/cli/utils/consoles.cpp:113]: No, it shouldn’t

**Pull request merged**